### PR TITLE
LE repo version bump

### PR DESCRIPTION
**Pull request by CvH**
@chewitt @jernejsk @Kwiboo @lrusak 
pls have a look if that changes for the addons are okay for your platforms

### CODE REVIEW

**Comment by lrusak** [comment]
Add-ons are built and in the repo? I'll just respin a test build and see if they work. Thanks!

**Comment by CvH** [comment]
> Add-ons are built and in the repo?
not yet depends at merge of https://github.com/LibreELEC/LibreELEC.tv/pull/4419

File: projects/Allwinner/options [hunk 1]
```diff
@@ -63,4 +63,4 @@
     DEBUG_TTY="/dev/console"
 
   # set the addon project
-    ADDON_PROJECT="H3"
+    ADDON_PROJECT="ARMv8"
```
Comment (jernejsk) [inline projects/Allwinner/options:66]: A20 and H3 are ARMv7, rest are ARMv8

**Comment by Kwiboo** [comment]
Similar to A20 and H3, RK3288 needs ARMv7

**Comment by CvH** [comment]
@Kwiboo pls have a look, changed it

**Pull request merged**